tests/cpp/test_multidevice_pipeline.cpp

```diff
@@ -576,6 +581,304 @@ INSTANTIATE_TEST_SUITE_P(
         SchedulingMode::Manual,
         SchedulingMode::ReductionOnly,
         SchedulingMode::Automatic));
+
+class DistributedMatmul : public MultiDeviceTest {
+ protected:
+  DistributedMatmul() : optimization_guard_(false) {
+    DisableOptionsGuard::getCurOptions().set(DisableOption::MatmulExprEval);
```

Comment:
There is a pathway to run matmuls through expression evaluator instead of applying the scheduler, so this flag disables that option to ensure the teset uses the scheduler.